python_modules/dagster/dagster/core/definitions/decorators/sensor.py

```diff
@@ -208,3 +214,80 @@ def _wrapped_fn(context, event):
         )
 
     return inner
+
+
+def multi_job_sensor(
+    jobs: List[Union[GraphDefinition, PipelineDefinition]],
```

Comment:
Nitpick: I've been moving to using `Sequence` instead of `List`, which is better because it's covariant and enforces that the parameter doesn't get mutated inside the function.